### PR TITLE
maintenance/prettier-formatting

### PR DESCRIPTION
**Pull request by daviduzumeri**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,8 +28,11 @@
     "@types/node": "12.12.14",
     "@types/react": "16.14.0",
     "docz": "2.3.1",
+    "husky": "4.3.7",
```
Comment (clayton-duarte) [inline package.json:31]: I used to use https://www.npmjs.com/package/pre-commit before knowing husky

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "build": "docz build",
     "start": "docz build && docz serve",
     "test": "jest --verbose --color --coverage",
+    "fmt": "prettier --write '{src,test}/**/*.{js,jsx,ts,tsx,css,json}'",
```
Comment (clayton-duarte) [inline package.json:9]: what `fmt` means?

Comment (daviduzumeri) [reply package.json:9]: `format`! It's what we use everywhere

File: src/components/Palette/index.spec.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ const paletteKeys = Object.keys(palette) as (keyof Theme['palette'])[]
 describe('Palette', () => {
   colorKeys.forEach((color) => {
     describe('should render a theme color, its value and name', () => {
-      it(`${color} >`, () => {
+      it(`${String(color)} >`, () => {
```
Comment (clayton-duarte) [inline src/components/Palette/index.spec.tsx:13]: I always thought that the template literal already converted the variables inside to string

Comment (daviduzumeri) [reply src/components/Palette/index.spec.tsx:13]: (I thought I removed this -- was a mistake on my part)

Comment (daviduzumeri) [reply src/components/Palette/index.spec.tsx]: Fixed!

**Review by clayton-duarte — APPROVED** [review]
Nice and tidy!

**Pull request merged**